lib/Dialect/FIRRTL/Export/FIREmitter.cpp

```diff
@@ -408,11 +409,25 @@ void Emitter::emitCircuit(CircuitOp op) {
   symInfos = std::nullopt;
 }
 
+void Emitter::emitEnabledLayers(ArrayRef<Attribute> layers) {
+  for (auto layer : layers) {
+    ps << PP::space;
+    ps.cbox(2, IndentStyle::Block);
+    ps << "enablelayer" << PP::space;
+    emitSymbol(cast<SymbolRefAttr>(layer));
+    ps << PP::end;
+  }
+}
+
 /// Emit an entire module.
 void Emitter::emitModule(FModuleOp op) {
   startStatement();
-  ps << "module " << PPExtString(legalize(op.getNameAttr())) << " :";
+  ps.cbox(4, IndentStyle::Block);
```

Comment:
I chose to double-indent continuations of the "module header" line, so that we can discern the module body from the continuation more easily, but I admit this is not as consistent as a single indent. It will print like this:
```
module Foo
    enablelayer LayerA
    enablelayer LayerB
    enablelayer LayerC :
  output o : UInt<1>
  input i : UInt<1>
   ```

Reply:
This subjectively looks better! Almost anything here is allowed by the spec, so taking some effort to make it pretty (within spec constraints) is great.